src/service/url-impl.js

```diff
@@ -141,6 +138,30 @@ export class Url {
   getWinOrigin(win) {
     return win.origin || this.parse(win.location.href).origin;
   }
+
+  /**
+   * If the current location is on a CDN, then convert the given resource
+   * URL to a path that resolves to the same resource served on the cache.
+   * @param {!Location|string} currentLocation The URL of the window
+   * @param {string} resourceUrl The URL of the document to load
+   * @return {string}
+   */
+  getCdnUrlOnCdnOrigin(currentLocation, resourceUrl) {
+    if (!isProxyOrigin(currentLocation)) {
```

Comment:
This is supposed to rewrite the URL if it's not on the cache, and do nothing if it is on the cache.

Reply:
I see, I got that backward